### PR TITLE
[iOS] Use the correct mins sim on 32b.

### PR DESCRIPTION
**Pull request by mandel-macaque**

### CODE REVIEW

**Review by akoeplinger — APPROVED** [review]

File: tests/Microsoft.DotNet.XHarness.iOS.Shared.Tests/Samples/simulators.xml [hunk 1]
```diff
@@ -60,15 +60,15 @@
...
+        <Identifier>com.apple.CoreSimulator.SimDeviceType.iPhone-5s</Identifier>
         <ProductFamilyId>IPhone</ProductFamilyId>
         <MinRuntimeVersion>393216</MinRuntimeVersion>
         <MaxRuntimeVersion>720895</MaxRuntimeVersion>
         <Supports64Bits>false</Supports64Bits>
       </SimDeviceType>
       <SimDeviceType>
         <Name>iPhone 5s</Name>
-        <Identifier>com.apple.CoreSimulator.SimDeviceType.iPhone-5s</Identifier>
+        <Identifier>com.apple.CoreSimulator.SimDeviceType.iPhone-5ss</Identifier>
```
Comment (mandel-macaque) [inline tests/Microsoft.DotNet.XHarness.iOS.Shared.Tests/Samples/simulators.xml:71]:
```suggestion
        <Identifier>com.apple.CoreSimulator.SimDeviceType.iPhone-5s</Identifier>
```

File: tests/Microsoft.DotNet.XHarness.iOS.Shared.Tests/Samples/simulators.xml [hunk 2]
```diff
@@ -453,104 +453,104 @@
...
-        <SimRuntime>com.apple.CoreSimulator.SimRuntime.iOS-10-3</SimRuntime>
+        <SimRuntime>com.apple.CoreSimulator.SimRuntime.iOS-11-3</SimRuntime>
         <SimDeviceType>com.apple.CoreSimulator.SimDeviceType.iPad-Pro--10-5-inch-</SimDeviceType>
         <DataPath>/Users/mandel/Library/Developer/CoreSimulator/Devices/356B263C-8516-4F3B-BBF1-CA23FA5478E6</DataPath>
         <LogPath>/Users/mandel/Library/Logs/CoreSimulator/356B263C-8516-4F3B-BBF1-CA23FA5478E6</LogPath>
       </SimDevice>
       <SimDevice UDID="711BA51B-6261-4394-84A6-BE9F8B5D3B80" Name="iPhone 5s">
         <SimRuntime>com.apple.CoreSimulator.SimRuntime.iOS-12-2</SimRuntime>
-        <SimDeviceType>com.apple.CoreSimulator.SimDeviceType.iPhone-5s</SimDeviceType>
+        <SimDeviceType>com.apple.CoreSimulator.SimDeviceType.iPhone-5ss</SimDeviceType>
```
Comment (mandel-macaque) [inline tests/Microsoft.DotNet.XHarness.iOS.Shared.Tests/Samples/simulators.xml:553]:
```suggestion
        <SimDeviceType>com.apple.CoreSimulator.SimDeviceType.iPhone-5s</SimDeviceType>
```

File: tests/Microsoft.DotNet.XHarness.iOS.Shared.Tests/Samples/simulators.xml [hunk 3]
```diff
@@ -453,104 +453,104 @@
...
+        <SimRuntime>com.apple.CoreSimulator.SimRuntime.iOS-11-3</SimRuntime>
+        <SimDeviceType>com.apple.CoreSimulator.SimDeviceType.iPhone-5s</SimDeviceType>
         <DataPath>/Users/mandel/Library/Developer/CoreSimulator/Devices/B0CF5BA5-318E-4FC8-8FC1-FB82D9F2FB42</DataPath>
         <LogPath>/Users/mandel/Library/Logs/CoreSimulator/B0CF5BA5-318E-4FC8-8FC1-FB82D9F2FB42</LogPath>
       </SimDevice>
       <SimDevice UDID="78A86F96-8333-42DE-81F3-1D8CFEC1FB17" Name="iPhone 5s">
-        <SimRuntime>com.apple.CoreSimulator.SimRuntime.iOS-10-3</SimRuntime>
-        <SimDeviceType>com.apple.CoreSimulator.SimDeviceType.iPhone-5s</SimDeviceType>
+        <SimRuntime>com.apple.CoreSimulator.SimRuntime.iOS-11-3</SimRuntime>
+        <SimDeviceType>com.apple.CoreSimulator.SimDeviceType.iPhone-5ss</SimDeviceType>
```
Comment (mandel-macaque) [inline tests/Microsoft.DotNet.XHarness.iOS.Shared.Tests/Samples/simulators.xml:463]:
```suggestion
        <SimDeviceType>com.apple.CoreSimulator.SimDeviceType.iPhone-5s</SimDeviceType>
```

**Comment by mandel-macaque** [comment]
Merging since android should not be affected by the change. @premun any idea of what happened?

**Pull request merged**